### ISSUE TITLE
Finishing a Process

### ISSUE THREAD

**Issue by MichalisKas** [FreedomCoop/valuenetwork#342]
There should be another step before finishing a process by clicking on 'Finish' button. Something like a screen to warn you that the process is going to be permanently deleted/dissapear and have the chance to Undo.

**Comment by fosterlynn** [comment]
The process should not permanently disappear, and you should see an "unfinish" button there..... provide a link?

**Comment by skoria** [comment]
It was a process about testing the carsharing app, which was created by gui, and clicked on by michelis

**Comment by MichalisKas** [comment]
I just noticed that if the creator of the process clicks on 'Finish' the there is an 'Unfinish' option.
So I propose to assign the 'Finish' button only to creators of the specific processes or admins.

**Comment by guyjames** [comment]
I agree it would be best to only let the creator of the process see the finish button. Also I would say 'Close' and 'Re-Open' are maybe better than 'Finish' and 'Un-Finish'...?

**Comment by MichalisKas** [comment]
Another thing we noticed is that even if the process is 'Finished', other participants can still log in work or make changes. I believe 'Finish' (or Close, or Complete) a process should lock it as well.
